api/server/router/local/image.go

```diff
@@ -289,12 +291,16 @@ func (s *router) postBuild(ctx context.Context, w http.ResponseWriter, r *http.R
 	defer output.Close()
 	sf := streamformatter.NewJSONStreamFormatter()
 	errf := func(err error) error {
+		errString := utils.GetErrorMessage(err)
+		if !buildConfig.Verbose {
+			errString = fmt.Sprintf("%s\n%s", notVerboseBuffer, errString)
```

Comment:
Like above, we can't combine the err with the output. In this case it results in the cli seeing something like:
```
Error response from daemon: Step 1 : FROM ubuntu
 ---> e9ae3c220b23
...
```
Where that "Error response from daemon:" text should be at the end when it shows the error text, not before the normal build output.